app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -446,13 +474,25 @@ private void raiseMessage(NotificationMessage message, boolean showTop) {
         }
         
         //either way
+        setLoginBoxesColor(R.color.cc_attention_negative_color, true);
+        username.setCompoundDrawablesWithIntrinsicBounds(getResources().getDrawable(R.drawable.icon_user_attnneg),  null, null, null);
+        password.setCompoundDrawablesWithIntrinsicBounds(getResources().getDrawable(R.drawable.icon_lock_attnneg),  null, null, null);
+        loginButton.setBackgroundColor(getResources().getColor(R.color.cc_attention_negative_bg));
+        loginButton.setTextColor(getResources().getColor(R.color.cc_attention_negative_text));
+
         errorBox.setVisibility(View.VISIBLE);
         errorBox.setText(toastText);
         
         Toast.makeText(this,toastText, Toast.LENGTH_LONG).show();
     }
-    
-    
+
+    private void setLoginBoxesColor(int colorId, boolean resolve) {

```

Comment:
'resolve' is an uninformative argument name and without method documentation, I have no idea what it is used for?